packages/node/test/unit/install.spec.ts

```diff
@@ -24,18 +28,28 @@ import {
   FreeBalanceStateJSON
 } from "../../src/models/free-balance";
 import { Store } from "../../src/store";
-import { EMPTY_NETWORK } from "../integration/utils";
 import { MemoryStoreService } from "../services/memory-store-service";
 
 import { createAppInstanceProposalForTest } from "./utils";
 
+const NETWORK_CONTEXT_OF_ALL_ZERO_ADDRESSES = EXPECTED_CONTRACT_NAMES_IN_NETWORK_CONTEXT.reduce(
```

Comment:
This should be in a more generic place as it's not specific to this test suite.

Reply:
It actually is specific to this test suite

Reply:
How? It's a network context object with all zero addresses. I don't see how it's specific to this test suite.

Reply:
It is never used anywhere else

Reply:
That's not a good reason to place a generic object in a test-specific context, but I digress.